src/Sylius/Bundle/VariationBundle/DependencyInjection/Configuration.php

```diff
@@ -115,7 +116,7 @@ private function addClassesSection(ArrayNodeDefinition $node)
                                             ->scalarNode('model')->isRequired()->cannotBeEmpty()->end()
                                             ->scalarNode('interface')->isRequired()->cannotBeEmpty()->end()
                                             ->scalarNode('controller')->defaultValue(ResourceController::class)->cannotBeEmpty()->end()
-                                            ->scalarNode('repository')->cannotBeEmpty()->end()
+                                            ->scalarNode('repository')->defaultValue(OptionRepository::class)->cannotBeEmpty()->end()
```

Comment:
This couples it to Doctrine ORM, we do not specify driver specific repositories in Configuration class. You should load a driver file, which will contain the repo. With VariationBundle it can be a bit more complex, because we generate the services, but I am pretty sure @lchrusciel will be able to help here because we have figured out this with ArchetypeBundle.